Homework2/StackCalculatorTests/CalculatorTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿using StackCalculator;
+
+namespace StackCalculatorTests;
+
+using System.Collections.Generic;
+using NUnit.Framework;
+
+[TestFixture]
+public class CalculatorTests
+{
+    private static IEnumerable<TestCaseData> StackCalulator
+        => new TestCaseData[]
+    {
+        new TestCaseData(new StackCalculator.StackCalculator(new StackCalculator.StackOnArray())),
+        new TestCaseData(new StackCalculator.StackCalculator(new StackCalculator.StackOnList())),
+    };
+
+    [TestCaseSource(nameof(StackCalulator))]
+    public void CalculateStandartCorrectPostfixSequence(StackCalculator.StackCalculator stackCalculator)
+    {
+        (float calculationResult, var isComputingCorrect) = stackCalculator.CalculateInPostfixForm("1 2 3 + *");
+        Assert.IsTrue(calculationResult == 5 && isComputingCorrect);
```

Comment:
```suggestion
        Assert.AreEquals(5, calculationResult)
        Assert.IsTrue(isComputingCorrect);
```
Всегда делите проверки на элементарные, чтобы если что-то не прошло, тут же было понятно, что не так.